backend/src/schemas/dtos.ts

```diff
@@ -0,0 +1,19 @@
+import { Types } from 'mongoose';
```

Comment:
Sometimes data payloads don't match the schemas or interfaces defined in our code. For example, data payloads for Device operations may need a `deviceToken` field, but the interface defining the schema contains a hashed token field.

The `dtos.ts` is useful for storing data payload objects that don't match or can't easily be derived from our schemas. IE the payload for the CRUD operation doesn't match the object fields.

Reply:
This isn't the way that nestjs recommends, as they have dto files for each module, but I thought that might be overkill for our use case.